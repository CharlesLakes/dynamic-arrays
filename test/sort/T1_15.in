10
78 93 40 13 12 75 50 22 2 51