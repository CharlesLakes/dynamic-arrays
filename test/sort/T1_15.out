2 12 13 22 40 50 51 75 78 93